packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -356,12 +358,11 @@ export class SmoothControls extends EventDispatcher {
         theta - clamp(deltaTheta, -dThetaLimit - dTheta, dThetaLimit - dTheta);
     const goalPhi = phi - deltaPhi;
 
-    const deltaRatio = deltaZoom === 0 ?
-        0 :
-        deltaZoom > 0 ? (maximumRadius! - radius) /
-                (Math.log(maximumFieldOfView!) - this.goalLogFov) :
+    const deltaRatio = deltaZoom === 0 ? 0 :
+        deltaZoom > 0                  ? (maximumRadius! - radius) /
+            (Math.log(maximumFieldOfView!) - this.goalLogFov) :
                         (radius - minimumRadius!) /
-                (this.goalLogFov - Math.log(minimumFieldOfView!));
+            (this.goalLogFov - Math.log(minimumFieldOfView!));
```

Comment:
This formatting looks weird; do you have clang-format set up?

Reply:
funny, it's not doing it for me. Well, they both seem equally weird, so no big deal. Honestly when the formatting gets this ugly it probably means we should just make some named `const`s and split this into multiple lines.